Shared/Samples/Edit features using feature forms/EditFeaturesUsingFeatureFormsView.swift

```diff
@@ -0,0 +1,205 @@
+                          let featureLayer = map.operationalLayers.first as? FeatureLayer else {
+                        return
+                    }
+                    defer { self.tapPoint = nil }
+                    
+                    do {
+                        let identifyLayerResult = try await mapViewProxy.identify(
+                            on: featureLayer,
+                            screenPoint: tapPoint,
+                            tolerance: 10
+                        )
+                        if let feature = identifyLayerResult.geoElements.first as? ArcGISFeature {
+                            featureLayer.selectFeature(feature)
+                            featureForm = FeatureForm(feature: feature)
+                            isShowingFeatureForm = true
+                        }
+                    } catch {
+                        self.error = error
+                    }
+                }
+                .overlay(alignment: .top) {
+                    Text(instructionText)
+                        .multilineTextAlignment(.center)
+                        .frame(maxWidth: .infinity, alignment: .center)
+                        .padding(8)
+                        .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
+                }
+                .floatingPanel(isPresented: $isShowingFeatureForm) { [featureForm] in
+                    if let featureForm {
+                        VStack {
+                            featureFormToolbar
+                            
+                            // Displays the feature form using the toolkit component.
+                            FeatureFormView(featureForm: featureForm)
+                                .padding(.horizontal)
+                                .task {
+                                    for await validationErrors in featureForm.$validationErrors {
+                                        hasValidationErrors = !validationErrors.isEmpty
+                                    }
+                                }
+                        }
+                    }
+                }
+                .alert("Discard Edits", isPresented: $isShowingDiscardEditsAlert) {
+                    Button("Cancel", role: .cancel) {
+                        isShowingDiscardEditsAlert = false
+                    }
+                    Button("Discard", role: .destructive) {
+                        isShowingDiscardEditsAlert = false
+                        featureForm?.discardEdits()
+                        endEditing()
+                    }
+                } message: {
+                    Text("Any changes made within the form will be lost.")
+                }
+                .errorAlert(presentingError: $error)
+        }
+    }
+    
+    /// The toolbar for the feature form panel.
+    private var featureFormToolbar: some View {
+        HStack {
+            Button("Discard Edits", systemImage: "trash", role: .destructive) {
```

Comment:
I think I prefer the "x" button as in the design screenshot.

The red trash 🗑️  icon gives me an impression that by clicking it, it would delete the feature from the feature table, which I don't want to happen, thus I don't dare to click it 😨